### PR TITLE
refactor: Moved A380X files into monorepo (no model)

### PR DESCRIPTION
**Pull request by frankkopp**
## Summary of Changes
This PR brings all a380x repo files into the aircraft repo (monorepo).
Only model, texture, sounds and effects remain in the a380x repo for now. 

Discord username (if different from GitHub): cdr_maverick

## Testing instructions
not required - should not impact the A32NX 

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/legacy/generated/A32NX_Exterior.xml [hunk 1]
```diff
@@ -0,0 +1,201 @@
+<ModelBehaviors>
```
Comment (tracernz) [inline fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/legacy/generated/A32NX_Exterior.xml:1]: Should the generated files be checked in? I suspect there's something wrong there.

Comment (frankkopp) [reply fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/legacy/generated/A32NX_Exterior.xml:1]: will fix

Comment (frankkopp) [reply fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/legacy/generated/A32NX_Exterior.xml:1]: > Should the generated files be checked in? I suspect there's something wrong there.

Are they even generated right now? Can't find a build script. Maybe leave them for now until these are really generated.

Comment (frankkopp) [reply fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/legacy/generated/A32NX_Exterior.xml:1]: Also there are A32NX
And there several A380_COCKPIT.xml etc. 
I think a good clean up after the move makes sense - from someone who understands all these files better ;)

Comment (frankkopp) [reply fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/legacy/generated/A32NX_Exterior.xml:1]: > Maybe somebody did a bit of a hack there.

Looks like Holland:
image

But also no changes ever - I do not think these are required any more. As I said - let's do a cleanup after the move.

Comment (Gurgel100) [reply fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/legacy/generated/A32NX_Exterior.xml:1]: IIRC there were some changes done in the generated xml files, which we have to carry over to the original files.

Comment (frankkopp) [reply fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/legacy/generated/A32NX_Exterior.xml:1]: > IIRC there were some changes done in the generated xml files, which we have to carry over to the original files.

Are these generated files even in use (referenced)? Or are they just to have the copy template?

Comment (tracernz) [reply fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/legacy/generated/A32NX_Exterior.xml:1]: > IIRC there were some changes done in the generated xml files, which we have to carry over to the original files.

Damn, that sounds even worse 😬

Comment (Gurgel100) [reply fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/legacy/generated/A32NX_Exterior.xml:1]: > > IIRC there were some changes done in the generated xml files, which we have to carry over to the original files.
> 
> Are these generated files even in use (referenced)? Or are they just to have the copy template?

They are used.

Comment (frankkopp) [reply fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/legacy/generated/A32NX_Exterior.xml:1]: > They are used.

Then we keep them for now of course until this is cleaned up.

Ready to merge if everybody else is good with it.

**Review by Benjozork — APPROVED** [review]

**Pull request merged**